mappings/net/minecraft/client/render/WorldRenderer.mapping

```diff
@@ -77,7 +77,7 @@ CLASS net/minecraft/class_761 net/minecraft/client/render/WorldRenderer
 		ARG 4 maxX
 		ARG 5 maxY
 		ARG 6 maxZ
-	METHOD method_21596 checkBlockRerender (Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;Lnet/minecraft/class_2680;)V
+	METHOD method_21596 scheduleBlockRerenderIfNecessary (Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;Lnet/minecraft/class_2680;)V
```

Comment:
```suggestion
	METHOD method_21596 scheduleBlockRerenderIfNeeded (Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;Lnet/minecraft/class_2680;)V
```
This name is a tiny bit long, needed shortening (no pun intended)

Reply:
Hmm, maybe `requestBlockRerender`? so if a block rerender is requested multiple times but not yet done it will be done just once, like multiple people request one same thing that thing will be done only once